From ae6d02e7e85c002e0326af8302574fff4b3a3f0a Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 19 Aug 2021 10:11:53 +0800
Subject: [PATCH] Revert "set --distinct_host_configuration=false by default
 for linux builds"

This reverts commit 0f9af911152880e36b2bfa8435e3746b9e329660.

Revert it to support cross compile

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 .bazelrc | 2 --
 1 file changed, 2 deletions(-)

diff --git a/.bazelrc b/.bazelrc
index 1544cf8892b..86a0f9a9c81 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -347,8 +347,6 @@ build:windows --verbose_failures
 
 # On windows, we never cross compile
 build:windows --distinct_host_configuration=false
-# On linux, don't cross compile by default
-build:linux --distinct_host_configuration=false
 
 # Configure short or long logs
 build:short_logs --output_filter=DONT_MATCH_ANYTHING
-- 
2.27.0

